packages/@okta/vuepress-site/docs/reference/user-query/index.md

```diff
@@ -0,0 +1,716 @@
+
+```JSON
+"id": "00u3q8ta4i7sbzIQv1d7",
+        "status": "ACTIVE",
+        "created": "2022-05-20T14:41:39.000Z",
+        "activated": "2022-05-20T14:41:41.000Z",
+        "statusChanged": "2022-05-20T14:41:41.000Z",
+        "lastLogin": null,
+        "lastUpdated": "2022-05-20T14:45:47.000Z",
+        "passwordChanged": "2022-05-20T14:41:40.000Z",
+        "type": {
+            "id": "oty2di92xFgQTiKEJ1d6"
+        },
+        "profile": {
+            "firstName": "Sylvia A.",
+            "lastName": "Ray",
+            "mobilePhone": null,
+            "customProp1": [
+                "a",
+                "b",
+                "c",
+                "d",
+                "e"
+            ],
+
+```
+
+#### Request example
+
+The following example returns all users who have a custom array value (`customProp1`) that includes at least one value equal to `a` or a custom array value (`customProp2`) that includes at least one value equal to `6`:
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users?search=profile.customProp1+eq+%22a%22+or+profile.customProp2+eq+%226%22"
+
+```
+
+#### Response example
+
+For brevity, only the user information and custom profile attribute are shown.
+
+```JSON
+{
+        "id": "00u3q8ta4i7sbzIQv1d7",
+        "status": "ACTIVE",
+        "created": "2022-05-20T14:41:39.000Z",
+        "activated": "2022-05-20T14:41:41.000Z",
+        "statusChanged": "2022-05-20T14:41:41.000Z",
+        "lastLogin": null,
+        "lastUpdated": "2022-05-20T14:45:47.000Z",
+        "passwordChanged": "2022-05-20T14:41:40.000Z",
+        "type": {
+            "id": "oty2di92xFgQTiKEJ1d6"
+        },
+        "profile": {
+            "firstName": "Sylvia A.",
+            "lastName": "Ray",
+            "mobilePhone": null,
+            "customProp1": [
+                "a",
+                "b",
+                "c",
+                "d",
+                "e"
+            ],
+            "customProp2": [
+                1,
+                2,
+                3,
+                4,
+                5
+            ],
+        }
+},
+{
+        "id": "00u3q8uarelmiiw0H1d7",
+        "status": "ACTIVE",
+        "created": "2022-05-20T14:42:13.000Z",
+        "activated": "2022-05-20T14:42:14.000Z",
+        "statusChanged": "2022-05-20T14:42:14.000Z",
+        "lastLogin": null,
+        "lastUpdated": "2022-05-20T14:46:54.000Z",
+        "passwordChanged": "2022-05-20T14:42:14.000Z",
+        "type": {
+            "id": "oty2di92xFgQTiKEJ1d6"
+        },
+        "profile": {
+            "firstName": "Bruce A.",
+            "lastName": "Ray",
+            "mobilePhone": null,
+            "customProp1": [
+                "f",
+                "g",
+                "h",
+                "i",
+                "j"
+            ],
+            "customProp2": [
+                6,
```

Comment:
Oh, and this really eats at me, so could you please use `7` instead of `6` in the example?  The thing that eats at me is that _every single one_ of your examples of using `eq` on an array turns out to match the _first_ element of the array.  Can we _please_ include at least one example where the matching element is _not_ the first?

Reply:
Done. Thanks.